Tests/MongoSwiftTests/CodecTests.swift

```diff
@@ -442,9 +442,13 @@ final class CodecTests: MongoSwiftTestCase {
     // test that Document.init(from decoder: Decoder) works with a non BSON decoder and that
     // Document.encode(to encoder: Encoder) works with a non BSON encoder
     func testDocumentIsCodable() throws {
+        // We presently have no way to control the order of emitted JSON in `cleanEqual`, so this
```

Comment:
honestly, I would be fine with just deleting this test. people should just do what I described in the "note" below if they want to convert between documents and JSON.

Reply:
I think we would like to eventually provide real `Codable` conformance for all our types, which is what this test is covering for `Document`. So I'd say it's probably worth leaving in if we expect it to be able to be run again in the future.

Reply:
in that case I would say let's just clean up the comments on why we are ignoring it, as it already says a few lines below that we are skipping on linux for key ordering reasons

Reply:
merged the two comments, removed the conditional OS check since we're skipping everywhere now.